<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncryptDialog</class>
 <widget class="QDialog" name="EncryptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>231</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Transaction Details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
   <property name="topMargin">
    <number>30</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Encrypt Wallet</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Please enter a passphrase to encrypt your wallet. This is required.
(You must use a minimum passphrase length of 10 characters and use uppercase letters, 
lowercase letters, numbers, symbols) </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="linePwd">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>New passphrase</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="linePwdConfirm">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Repeat new passphrase</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>100</number>
     </property>
     <property name="leftMargin">
      <number>100</number>
     </property>
     <property name="rightMargin">
      <number>100</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>Confirm</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
